### ISSUE TITLE
you should probably consider taking this down

### ISSUE THREAD

**Issue by investreads** [anderly/piklist-field-addons#3]
If i added up all the time wasted on busted Github code...it would be a significant amount of time. In the words of Spike Lee, do the right thing.

**Comment by anderly** [comment]
Yep, it’s an old repo. A quick check of the last commit date would tell you that and save you the headache. I leave it up as a reference. I’ll consider adding a note to reflect it’s no longer current.
